ray/operator/scc/ray_operator_scc.yaml

```
@@ -0,0 +1,10 @@
+kind: SecurityContextConstraints
+metadata:
+  name: run-as-ray-user
+seLinuxContext:
+  type: MustRunAs
+runAsUser:
+  type: MustRunAs
+  uid: 1000
+users:
+  - 'system:serviceaccount:opendatahub:kuberay-operator'
```

```suggestion
  - 'system:serviceaccount:<deployed-namespace>:kuberay-operator'
```

I need help getting the namespace to be based on where the kuberay operator is actually deployed.

---

Just using the var ref works fine. It doesn't have to replace the whole value as I thought that it did.